build_tools/cmake/build_all.sh

```diff
@@ -56,7 +56,9 @@ declare -a CMAKE_ARGS=(
   "-DIREE_BUILD_PYTHON_BINDINGS=ON"
   "-DPython3_EXECUTABLE=${IREE_PYTHON3_EXECUTABLE}"
   "-DPYTHON_EXECUTABLE=${IREE_PYTHON3_EXECUTABLE}"
-
+  "-DIREE_USE_SCCACHE=1"
```

Comment:
Drive-by comments:

* You don't need to create a pull request to use `workflow_dispatch`. If your work is still a draft and you aren't yet looking for comments, then you can limit notification noise by just pushing to your branch and triggering.
* I expect that configuring and debugging sccache [in scripts] locally on Windows will be easier than iterating on the github actions workflow. If you've already done that, then great.
* You may want to issue calls directly instead of using `build_all.sh`, or write a new `build_all_windows.sh`. The script (https://github.com/iree-org/iree/blob/main/build_tools/cmake/build_all.sh) is mostly plumbing and configuration, which you can bypass when writing a workflow directly. See what the other workflows do, for example: https://github.com/iree-org/iree/blob/d4975713a0aa3ba872807fc16585fb4b5a04e41d/.github/workflows/ci_linux_x64_clang.yml#L50-L68 (actually, now that I look at that, I think some workflows are missing `-DIREE_TARGET_BACKEND_CUDA=ON` and `-DIREE_TARGET_BACKEND_ROCM=ON` :thinking: )

Reply:
Sorry, I thought it said creating a draft PR didn't send out emails